tests/api_resources/test_customer.py

```diff
@@ -197,7 +197,7 @@ def test_is_listable(self, http_client_mock):
     def test_is_listable_on_object(self, http_client_mock):
         resource = stripe.Customer.retrieve(
             TEST_RESOURCE_ID
-        ).list_payment_methods(TEST_RESOURCE_ID, type="card")
+        ).list_payment_methods(type="card")
```

Comment:
What happened here?

Reply:
This was never actually supported syntax since it's being called on a customer instance - `TEST_RESOURCE_ID` was being use as the `idempotency_key` here. Caught this by changing request options to be keyword-only.